kolibri/plugins/learn/assets/src/views/QuizRenderer/QuizReport.vue

```diff
@@ -17,6 +17,7 @@
     :questions="questions"
     :isSurvey="isSurvey"
     :isQuiz="!isSurvey"
+    :sections="exam.question_sources"
```

Comment:
This will never be defined - can be cleaned up once there's a default on the sections prop.